Backend/src/modules/users/users.controller.ts

```diff
@@ -42,4 +43,44 @@ export default class UserController extends Api {
       );
     }
   };
+
+  public UpdateUserInfo = async (
+    req: Request<unknown, unknown, UpdateUserRequestDto>,
+    res: Response
+  ) => {
+    try {
```

Comment:
you don't need try-catch except if you want to do special logic (e.g. retrieve from cache/db)

Our expressjs has a middle ware that log all thrown errors and return the appropriate response.